### PR TITLE
Fixing other issues encountered during testing

### PR DESCRIPTION
**Pull request by MV88**
## Description

- 3d
- search by bookmark
- geostory
- dashboard embedded
- permalink
- dashboard catalog service

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#<issue>

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

**Review by tdipisa — CHANGES_REQUESTED** [review]
@MV88 I'm sorry I don't see permalink enabled in this config.

**Comment by tdipisa** [comment]
> @MV88 I'm sorry I don't see permalink enabled in this config.

already added before same for geostory it seems

File: configs/localConfig.json [hunk 1]
```diff
@@ -823,7 +826,7 @@
           "services": {
             "PubliAcqua GeoServer WMS": {
               "url": "https://web-gis.publiacqua.it/geoserver/wms",
```
[the remark this answers is not in the document]
Comment (MV88) [reply configs/localConfig.json:828]: ok, before it was a mix so i decided to go on wms, but csw should be available

[the remark this answers is not in the document]
Comment (MV88) [reply configs/localConfig.json:828]: not available 
image

**Review by tdipisa — APPROVED** [review]

**Pull request merged**